hack/perf/bz-2162111.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env sh
+source ./helpers.bash
+
+tmp=$(mktemp -d)
+
+file_a=$(mktemp -p $tmp --suffix '.file_a')
+file_b=$(mktemp -p $tmp --suffix '.file_b')
+dd if=/dev/zero of=$file_a bs=1024 count=1024 status=none
+dd if=/dev/zero of=$file_b bs=1024 count=1024 status=none
+
+# The create command
+volume_name="bz-2162111"
+container_name="bz-2162111"
+container_cmd="--name $container_name \
+	--stop-timeout=0 \
+	--network-alias alias_a \
+        --network-alias alias_b \
```

Comment:
mini nit: indentation is off

Reply:
Can you elaborate? Aren't they all on tab?

Reply:
Ah, got it. Nice catch, thank you!